blackbox/docs/sql/administration/user_defined_functions.txt

```diff
@@ -232,10 +232,72 @@ the JavaScript function should return a ``GeoJson`` object or``WKT`` string::
 
 .. NOTE::
 
-   If the return value of the JavaScript function is ``undefined`` it is converted
-   to ``NULL``.
+   If the return value of the JavaScript function is ``undefined`` it is
+   converted to ``NULL``.
 
 
+Working with ``TIMESTAMP``
```

Comment:
i would rename it to, Working with ``NUMBERS``. Although it's only a problem when it's used in combination with timestamps, it's a number specific problem and might be interesting for not only those who deal with timestamps.